idb/postgres/internal/testing/testing.go

```diff
@@ -14,15 +15,23 @@ import (
 	"github.com/algorand/indexer/idb/postgres/internal/schema"
 )
 
-var testpg = flag.String(
-	"test-pg", "", "postgres connection string; resets the database")
+var testpg string
+
+func init() {
+	flag.StringVar(&testpg, "test-pg", "", "postgres connection string; resets the database")
+	if testpg == "" {
+		// Note: tests across packages run in parallel, so this does not work
+		// very well unless you use "-p 1".
+		testpg = os.Getenv("TEST_PG")
+	}
+}
```

Comment:
does the process exit with error if testpg is empty, which happens if neither test-pg or TEST_PG is set?

Reply:
If TEST_PG isn't set it returns an empty string, so it would be a no-op.